sympy/core/evalf.py

```diff
@@ -1275,6 +1275,7 @@ def _create_evalf_table():
     from sympy.functions.elementary.piecewise import Piecewise
     from sympy.functions.elementary.trigonometric import atan, cos, sin
     from sympy.integrals.integrals import Integral
+    from sympy.integrals.risch import NonElementaryIntegral
```

Comment:
Hi @sidhu1012 !

Because of this, the following [test fails now](https://github.com/sympy/sympy/blob/4c87ed427cc239687fd7c7fe4bf72d1d2ab0aae7/sympy/core/sympify.py#L185) (confirmed by the [test output here](https://github.com/sympy/sympy/pull/20481/checks?check_run_id=1446919038#step:6:475) ).

This may be either because of [this line](https://github.com/sympy/sympy/blob/4c87ed427cc239687fd7c7fe4bf72d1d2ab0aae7/sympy/abc.py#L71) or [this line](https://github.com/sympy/sympy/blob/4c87ed427cc239687fd7c7fe4bf72d1d2ab0aae7/sympy/abc.py#L71).

Either way, it seems like the symbol **C** is being introduced, and because of that, the test mentioned at the top of the comment will fail.

I wonder if adding the **C** symbol to the expected dictionary data in the test would fix this?